SwiftWisdomTests/StandardLibrary/Collection/Collection+RandomTests.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  Collection+RandomTests.swift
+//  SwiftWisdom
+//
+//  Created by Andrew Dolce on 11/21/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import XCTest
+@testable import SwiftWisdom
+
+class CollectionRandomTests: XCTestCase {
+
+    // This test is complex. Here is an explanation as to why:
```

Comment:
I do think these tests are quite comprehensive but for the reasons that you have mentioned, i kind of think that we should not include them...they seem unwieldy to update and seem to be dancing around the 'actual' random test.

Fwiw, I ran this code:

```
var array = repeatElement(0, count: Int(UInt8.max)).ip_toArray()

        for _ in 0...1000000 {
            let index = UInt8.ip_random()
            array[Int(index)] = array[Int(index)] + 1
        }
        print(array)
```

and got

```
[3927, 3857, 3833, 3904, 3933, 3932, 3956, 3889, 4012, 3949, 3924, 3888, 3871, 3890, 3891, 3900, 4003, 3945, 3804, 3898, 3934, 4019, 3976, 4001, 3885, 3893, 3940, 4007, 3872, 4012, 3954, 3971, 3829, 4060, 3953, 3850, 4010, 3873, 3963, 3979, 4003, 4006, 3922, 3836, 3893, 3958, 3799, 3873, 3894, 3976, 3840, 3890, 3964, 3916, 3917, 3978, 3918, 3816, 3925, 3880, 3974, 3914, 4006, 3945, 3748, 3888, 4020, 3876, 3981, 3994, 3941, 3809, 3962, 3929, 3922, 3854, 3928, 3940, 3978, 3930, 3912, 3864, 3818, 3983, 3895, 3869, 3859, 3972, 3972, 3905, 3948, 3992, 4049, 3936, 3914, 3895, 3931, 3985, 3882, 3895, 3926, 4041, 3865, 4000, 3965, 3987, 3853, 3987, 4011, 3823, 3815, 3902, 3968, 3940, 3818, 3866, 4001, 3974, 3866, 3980, 3952, 4053, 3890, 4031, 3896, 3843, 3857, 3893, 3905, 3967, 4091, 4035, 3918, 3881, 3826, 3955, 3898, 3896, 3854, 3966, 3854, 4006, 4012, 3878, 3957, 3914, 3922, 3934, 3885, 4037, 3962, 3937, 3848, 3920, 3834, 3860, 3859, 3858, 3966, 3900, 3948, 3929, 3856, 3925, 3824, 3888, 3934, 3848, 3980, 3935, 3942, 3923, 3934, 3871, 3886, 3867, 3886, 3904, 3886, 3960, 3848, 3890, 3871, 4010, 3846, 3962, 3919, 3897, 3884, 3897, 3953, 3901, 3826, 3899, 4008, 3965, 4018, 3999, 3870, 3938, 3853, 3887, 3948, 3889, 3860, 3951, 3920, 3882, 3916, 3913, 3875, 4023, 3944, 3956, 3851, 3921, 3926, 4018, 3916, 3955, 3910, 3842, 3908, 3915, 3935, 3899, 4005, 3975, 3947, 3884, 3869, 3983, 3831, 3931, 3926, 3897, 3927, 3980, 3937, 3953, 3927, 3889, 3849, 3981, 3929, 3929, 3943, 3872, 3993, 3902, 3837, 4080, 3837, 3894, 3810]
```

Not by any means a scientific test, but I feel good about it.